jobs/cloud_controller_ng/spec

```diff
@@ -1002,6 +1002,9 @@ properties:
   uaa.clients.cloud_controller_username_lookup.secret:
     description: "Used for fetching usernames from UAA"
 
+  uaa.clients.cloud_controller_shadow_user_creation.secret:
```

Comment:
Will you create a PR to add the new secret to cf-deployment? Similar to e.g.:
https://github.com/cloudfoundry/cf-deployment/blob/0aeb4407b801212c09d716bf2c7814306364b5b3/cf-deployment.yml#L2043

Reply:
We might add an ops file later. But this secret is not required -> won't add it by default to cf-deployment.